src/common/Journald.cc

```diff
@@ -3,7 +3,6 @@
 
 #include "Journald.h"
 
-#include <endian.h>
```

Comment:
?

Reply:
should I make an `endian-compat.h` instead?